src/net/peacefulcraft/sco/gamehandle/player/SCOPlayer.java

```diff
@@ -108,18 +109,22 @@ public void removeExhaustion(double exhaustion) {
 	private List<Modifier> damageModifiers;
 		public List<Modifier> getDamageModifiers() { return this.damageModifiers; }

```

Comment:
Wished I made this realization more recently than a few weeks ago, but when we have getters for List and Maps, we should either be copying them and returning the copies, or returning [unmodifiable wrappers](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html), otherwise the property might as well be `public` because you can call `hypotheticalGetter().add() or .remove()` and it will change the list / map. 

Technically a potential bug or security issue with any Object getter because pass by reference, but Lists and HashMaps are usually tracking things internal to a class that we don't expect to be modified outside that class. Could cause obscure bugs in 6 months when we forget where something is populating from.

Actual fix:
```Java
public List<Modifier> getDamageModifiers() { return Collections.unmodifiableList( this.damageModifiers ); }
```